scripts/code_analyze.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh -ve
 flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
 dart format --set-exit-if-changed lib/ test/
-flutter analyze
+dart run dart_code_metrics:metrics check-unused-files lib
```

Comment:
Please note, `dart-code-metrics` is discontinued and now requires a license: https://github.com/dart-code-checker/dart-code-metrics.

Reply:
Not that I know of.

Reply:
dont know another lib
also this check cannot ignore files
so this PR is pending